charts/osdfir-infrastructure/README.md

```diff
@@ -249,23 +282,33 @@ through dynamic provisioning.
 The volumes for ArangoDB and Redis automatically get created during deployment
 through dynamic provisioning.
 
+#### Persistence in OpenRelik
+
+By default, OpenRelik mounts a Persistent Volume at the
+`/mnt/openrelik` path to store uploaded output for OpenRelik to process.
```

Comment:
you reference /mnt/openrelikvolume instead in config parameters.

Reply:
Good catch, updated to `/mnt/openrelikvolume`